internal/plugin/subprocess_commands_test.go

```diff
@@ -224,16 +225,18 @@ func TestPrepareCommandsNoCommands(t *testing.T) {
 }
 
 func TestPrepareCommandsExpand(t *testing.T) {
-	t.Setenv("TEST", "test")
+	t.Setenv("TESTX", "testx")
```

Comment:
minor: Can we just move this logic to be included below?:

```
env["TESTX"] = "testx"
```

(as the usage of `parseEnv` is just causing this to round-trip through the process env vars)